mne/io/egi/egimff.py

```diff
@@ -495,8 +514,22 @@ def __init__(self, input_fname, eog=None, misc=None,
         info['chs'] = chs
         info._unlocked = False
         info._update_redundant()
+
         if mon is not None:
             info.set_montage(mon, on_missing='ignore')
+
+        ref_idx = np.flatnonzero(np.in1d(mon.ch_names, REFERENCE_NAMES))
+        if len(ref_idx):
+            ref_coords = info['chs'][int(ref_idx)]['loc'][:3]
+            for chan in info['chs']:
+                is_eeg = chan['kind'] == FIFF.FIFFV_EEG_CH
+                is_not_ref = chan['ch_name'] not in REFERENCE_NAMES
+                if is_eeg and is_not_ref:
+                    chan['loc'][3:6] = ref_coords
+
+            # Cz ref was applied during acquisition, this will mark as already set.
```

Comment:
```suggestion
            # Cz ref was applied during acquisition, so mark as already set.
```
(just to get it below 80 chars)